pin-devices-targeting-release.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+## This script sets devices targeting a given commit to target a new commit
+## Usage: ./pin-devices-to-target-release <FULL_COMMIT_HASH> <NEW_FULL_COMMIT_HASH>
```

Comment:
Would it make more sense if we used a slightly different name like `./update-pinned-devices` or `./update-devices-pinned-to-release`?
My only point is that `./pin-devices-to-target-release` doesn't describe that the target group is a set of already pinned devices.